### DISCUSSION TITLE
One-hand-use Concept

### DISCUSSION THREAD

**Discussion by JVariance** [holisticapps/Noteful-Issues#6] [Show and tell]
I made a concept for a one handed use optimized UX (in this case as PWA).
All elements are easily reachable even if the phone is used with just one hand.

https://user-images.githubusercontent.com/62245444/126879813-448743f9-6ef3-4f25-9f71-013182c23b42.mp4

**Comment by tschiekdev** [comment]
Hm, the problem would basically also affect all the other apps on the smartphone. If that bothers you, then you should buy a smaller smartphone, right? For this reason, for example, I only use smaller devices (5.5 inch Pixel 3a).

**Reply by JVariance** [reply]
I have a 5.1 inch Galaxy S7 and stick to it because it's one of the smallest smartphones😃
And it's true that it's the case for all apps, but should this be the argument for not further improving the layout & flow?
I use Samsungs app 'One Hand Operation+' for dragging the screen down, which let's me access elements from the top of the screen, but this kinda breaks the flow.

Two nice examples for a drag up menu are Google Maps and the EDEKA app.
The EDEKA app has a bottom menu which includes a button to open/automatically slide up the menu. Or you can drag it up manually. I love such fluent and accessible experiences.

Samsung does a good job in this regard with the newer One UIs.
For example:
![grafik](https://user-images.githubusercontent.com/62245444/127863284-f7b3d478-246a-47c0-8326-5210ee327ee4.png) copied from [this site](https://www.neowin.net/news/top-samsung-one-ui-30-and-android-11-features/).
